### DISCUSSION TITLE
how to config ssl in url, is any demo for pymysql ssl connection

### DISCUSSION THREAD

**Discussion by stevenxuwoss** [agronholm/sqlacodegen#166] [Q&A] [answered]
not issue just question.

**Comment by agronholm** [comment]
Is [this](https://docs.sqlalchemy.org/en/14/dialects/mysql.html#mysqldb-ssl) helpful?

**Comment by stevenxuwoss** [comment, marked as answer]
i found a workaround to use mysqlconnector not pymysql, it can solve my issue

**Reply by agronholm** [reply]
What was the reason you were unable to use SSL with pymysql?

**Reply by stevenxuwoss** [reply]
my issue is there's need to enable ssl but no ca and key. i don't know how to set this ssl to true in url with pymysql.

**Reply by stevenxuwoss** [reply]
if config ca and key, i can connect with pymysql. but now the server no config ca and key just enable ssl,
<img width="588" alt="1637845973242" src="https://user-images.githubusercontent.com/17000812/143447867-b3d516e2-8b08-4e2e-9241-1fd02532226a.png">

**Reply by agronholm** [reply]
PyMysql's documentation is also a bit unclear about that. Would `?ssl_disabled=false` work?

**Reply by stevenxuwoss** [reply]
i tried this, not work

**Reply by agronholm** [reply]
Ah, well. Glad you at least got it to work with mysqlconnector.
